extra/lempar.c

```diff
@@ -451,8 +451,7 @@ static unsigned int yy_find_shift_action(
   int i;
   int stateno = pParser->yytos->stateno;
  
-  if( stateno>=YY_MIN_REDUCE ) return stateno;
-  assert( stateno <= YY_SHIFT_COUNT );
+  if( stateno>YY_SHIFT_COUNT ) return stateno;
```

Comment:
How about adding `assert(stateno >= YY_MIN_REDUCE || stateno <= YY_SHIFT_COUNT);` to the previous line?

Reply:
done.